tgui/packages/tgui/interfaces/MedicalRecords.js

```diff
@@ -109,27 +120,77 @@ export const MedicalRecords = (_properties, context) => {
 const MedicalRecordsList = (_properties, context) => {
   const { act, data } = useBackend(context);
   const { records } = data;
+  const [searchText, setSearchText] = useLocalState(context, 'searchText', '');
+  const [sortId, _setSortId] = useLocalState(context, 'sortId', 'name');
+  const [sortOrder, _setSortOrder] = useLocalState(context, 'sortOrder', true);
   return (
-    <Fragment>
-      <Input
-        fluid
-        placeholder="Search by Name, DNA, or ID"
-        onChange={(_event, value) => act('search', { t1: value })}
-      />
-      <Box mt="0.5rem">
-        {records.map((record, i) => (
-          <Fragment key={i}>
-            <Button
-              icon="user"
-              mb="0.5rem"
-              content={record.id + ': ' + record.name}
-              onClick={() => act('d_rec', { d_rec: record.ref })}
-            />
-            <br />
-          </Fragment>
-        ))}
-      </Box>
-    </Fragment>
+    <Flex direction="column" height="100%">
+      <Flex>
```

Comment:
did you mean to have two `<Flex>` elements here?

Reply:
Yep, it prevents some weird stuff with the search bar and having buttons next to it.